ui/raidboss/data/04-sb/dungeon/temple_of_the_fist.js

```diff
@@ -50,5 +121,202 @@
         fr: 'trouvez une zone safe',
       },
     },
+    {
+      id: 'Temple Spirit Wave',
+      regex: / 14:1FE7:Ivon Coeurlfist starts using Spirit Wave/,
+      condition: function(data) {
+        return data.role == 'healer';
+      },
+      infoText: {
+        en: 'AoE',
+      },
+    },
+    {
+      id: 'Temple Touch Of Slaughter',
+      regex: / 14:1FE6:Ivon Coeurlfist starts using Touch Of Slaughter on (\y{Name})/,
+      alertText: function(data, matches) {
+        if (data.me == matches[1]) {
+          return {
+            en: 'Slaughter on YOU',
+          };
+        } else if (data.role == 'healer') {
+          return {
+            en: 'Heal ' + data.shortName(matches[1]) + ' soon',
+          };
```

Comment:
While not a bad alert, I don't think we have anything like it currently. Does it make more sense for the move to resolve and _then_ alert? Or not alert at all, since the healer should have ample time to throw out a heal even if only reacting to mechanics as they happen?

Not that new triggers are bad, it just made me do a double-take. Feel free to ignore this one!

Reply:
I'm not sure. The healer is guaranteed to be required to heal them, and we already pre-warn on tank busters and party AoE damage, so I feel like it's not terribly different in that sense. Obviously it's not a party-wipe level of danger like busters, but the logic feels the same to me.